grails-app/views/inventoryItem/_transferStock.gsp

```diff
@@ -69,12 +69,9 @@
                     ${location.name}
                 </td>
                 <td>
-                    <g:selectLocation id="transferTo-${inventoryItem?.id}" name="otherLocation.id" class="chzn-select-deselect trigger-change"
-                                      noSelection="['null':'']"
-                                      data-placeholder="${g.message(code: 'inventoryItem.chooseStockTransfer.label', default: 'Choose where stock is being transferred to ...')}"
-                                      data-url="${request.contextPath}/inventoryItem/refreshBinLocation"
-                                      data-target="#transferTo-binLocation"
-                                      value=""/>
+                    <g:selectTransactionDestination name="otherLocation.id" class="chzn-select-deselect trigger-change"
```

Comment:
What about refreshing the bin locations?

Reply:
thanks for noticing, I amended the commit and probably did a `git push` which failed, instead of a `git push --force`